src/docs/markdown/caddyfile/directives/bind.md

```diff
@@ -64,6 +67,39 @@ example.com {
 }
 ```
 
+To bind to a Unix domain socket at `/run/caddy/stream.sock` that serves h1 and h2, and another at `/run/caddy/dgram.sock` that serves h3:
+
+```caddy
+example.com {
+	bind unix//run/caddy/stream.sock {
+		protocols h1 h2
+	}
+	bind unixgram//run/caddy/dgram.sock {
+		protocols h3
+	}
+}
+```
+
+To bind to inherited file descriptors specified with [environment placeholders](/docs/conventions#placeholders):
```

Comment:
I think this should explain a bit why you'd want to use file descriptors. As a reader, I think "but why?"